1-Draft/RFCXXXX-Obsoletion-and-Deprecation.md

```diff
@@ -0,0 +1,511 @@
+              Message = _string_ # Optional - The message to omit when the module is loaded.
+              MemberSignature = _string_ # Optional - used only to disambiguate multiple member names. If omitted, all members of this name will be assumed obsolete.
+              MemberType = _string_ # Property|Field|Method|Constructor - Optional - used only disambiguate multiple members. If omitted, all members with the name will be assumed obsolete.
+              Action = _SilentlyContinue_|_Continue_|_Stop_|_NotFound_ # Optional - This is the action to take. Default behavior is "continue" which will produce a warning.
+            }
+          )
+        }
+    )
+}
+```
+
+The following is an example from the Microsoft.PowerShell.Utility module:
+
+```powershell
+...
+PrivateData = @{
+  PSData = @{
+    ExperimentalFeatures = @(
+      @{
+        Name        = 'Microsoft.PowerShell.Utility.PSManageBreakpointsInRunspace'
+        Description = 'Enables -BreakAll parameter on Debug-Runspace and Debug-Job cmdlets to allow users to decide if they want PowerShell to break immediately in the current location when they attach a debugger.'
+      }
+    )
+    ObsoleteHints = @{
+      Commands = @{
+        CommandName = "Format-Hex"
+        IsObsolete = $false
+        Parameters = @{ ParameterName = "Raw"; Message = "Raw parameter is obsolete." }
+      },
+      @{
+        CommandName = "Send-MailMessage"
+        Message = "This cmdlet does not guarantee secure connections to SMTP servers. While there is no immediate replacement available in PowerShell, we recommend you do not use Send-MailMessage at this time. See https://aka.ms/SendMailMessage for more information."
+        Action = "NotFound"
+      }
+    }
+  }
+}
+...
+```
+
+- the MemberSignature is for disambiguation of obsolescence for the member of a type which may have overloaded signatures.
+- If there are no obsolete types, then `Types` may be omitted.
+- If there are no obsolete commands (cmdlets, functions, or scripts), then `Commands` may be omitted.
+- If there are no obsolete parameters, then `Parameters` may be omitted.
+- If there are no obsolete members, then `Members` may be omitted.
+
+The actual `ObsoleteAttribute` must still be present in the command or type.
+The source of truth once the module is imported is the presence of the `ObsoleteAttribute` in the source of the cmdlet, parameter, type, or member.
+The elements are present in the manifest to not require that the module is loaded before being able to report on obsolescence.
+
+## Configuration Tools
+
+There are a number of elements that require configuration for managing the behavior when an element is marked for obsolescence.
+Because of this, we shall provide cmdlets to aid with the creation of the configuration.
+Three cmdlets, `Get-ObsoleteItem`, `Enable-ObsoleteItem`, and `Disable-ObsoleteItem` shall be provided.
+
+### `Get-ObsoleteItem`
```

Comment:
I wonder if `ObsoleteFeature` would be a better noun.  We have `ExperimentalFeature` which applies to many of the same granularities as obsolescence